vm/devices/storage/disk_nvme/nvme_driver/src/driver.rs

```diff
@@ -543,8 +806,11 @@ impl<T: DeviceBacking> DriverWorkerTask<T> {
             return;
         }
 
+        // Provide next available queue ID.
+        let qid = self.io.len() as u16 + 1;
```

Comment:
It seems like this change wasn't actually necessary.

Reply:
Let me double check. This might be another leftover from previous logic when I was slicing one memory into multiple chunks.